packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -0,0 +1,3 @@
+from pkgpanda.util import load_json
+
+dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
-Is it possible to have a master node that is considered "healthy" that doesnt have an expanded config? I do not think so. Given that, we already have made it a requirement that we run tests on masters so I do not see an issue here
-there is no such thing as a default provider and dns_search defaults to None. Could add some default None but then that gets confused with when the actual config is None. I would rather hard error on the config not being present that be ambiguous. Same goes for adminrouter auth state, default is enabled, but I would rather error hard if the util needs to guess. Also, there will be an endpoint to get security type, so we will probably switch to using that at some point.